conf/application.conf

```diff
@@ -78,15 +78,7 @@ microservice {
       fetch-notifications = "/notifications"
       fetch-submissions = "/submissions"
       cancel-declaration = "/cancellations"
-    }
-
-    customs-declarations-information {
-      host = localhost
-      port = 9834
-      fetch-mrn-status = "/mrn/ID/status"
-      api-version = "1.0"
-      bearer-token = "Bearer customs-declare-exports-frontend"
-      client-id = "customs-declare-exports-frontend"
+      fetch-ead = "/ead"
```

Comment:
Probably just naming, because we're not fetching the EAD but the dec information. Of course it's just for EAD, so I think we can stay with this

Reply:
But after checking backend, probably it's okay to stay with this naming

Reply:
Let's continue with this and then discuss once we break the microservice